### PR TITLE
Upgrade Whiskey

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: whiskey.yml [hunk 1]
```diff
@@ -1,31 +1,58 @@
...
+- master
+- prerelease
+
 Build:
 - Version:
     Path: Glob\Glob.psd1
+    Prerelease:
+    - prerelease: beta$(WHISKEY_BUILD_NUMBER)
+
+# Dot-sourcing a lot of files when importing a module varys from expensive to really expensive. Merge them into the module manifest.
```
Comment (decoyjoe) [inline whiskey.yml:12]: Typo: `varys` should be `varies`

File: whiskey.yml [hunk 2]
```diff
@@ -1,31 +1,58 @@
...
+    - Glob\Functions\*.ps1
+    DestinationPath:
+    - Glob\Glob.psm1
+    TextSeparator: "$(NewLine)$(NewLine)"
+    DeleteSourceFiles: true
 
 - Pester4:
     Path: Tests\*.Tests.ps1
+    Verbose: false
+    Debug: true
```
Comment (decoyjoe) [inline whiskey.yml:25]: `Debug: true` intentional?

**Review by decoyjoe — APPROVED** [review]

File: whiskey.yml (hunk 2 above)
Comment (splatteredbits) [reply whiskey.yml:25]: Fixed in the other branch.

File: whiskey.yml (hunk 1 above)
Comment (splatteredbits) [reply whiskey.yml:12]: I'll fix in the other branch.

**Pull request merged**